tests/Doctrine/Tests/OrmFunctionalTestCase.php

```diff
@@ -1051,6 +1060,11 @@ final protected function getLastLoggedQuery(int $index = 0): array
         return $lastQuery;
     }
 
+    final protected function assertQueryLog(string ...$expectedQueries): void
+    {
+        self::assertSame($expectedQueries, array_column($this->getQueryLog()->queries, 'sql'));
+    }
```

Comment:
Introducing that method allows to compare the 2 query logs rather than getting an exception because there is no seventh query.